hasher-matcher-actioner/src/OpenMediaMatch/templates/components/add_to_bank_form.html.j2

```diff
@@ -14,10 +14,11 @@
         </div>
         <div class="col">
             <select class="form-select" name="content_type" required>
+                <option selected disabled value="">Select content type</option>
                 {% for c in content %}
-                {% if c['enabled'] %}
-                <option value="{{c['name']}}">{{c['name'].capitalize()}}</option>
-                {% endif %}
+                    {% if c['enabled'] %}
```

Comment:
I found a library to auto-format these, I should add it to the devcontainer so we can just let it handle it.